dev-env/README.md

```diff
@@ -1,17 +1,38 @@
 # dev-env
 
 This directory contains supporting files for the included MetalLB development
-environment. The environment is run using the following command:
+environment. The environment is run using the following command from the root of your git clone:
 
 ```
 inv dev-env
```

Comment:
This change is to add additional comment that user should execute inv command at the top of repo directory not dev-env/ directory because user tend to execute command from the same path of the document, so it is just additional advice.

Add some text but I could revert if it is not still clear. This is not so important.